### PR TITLE
Adds DRUSH_OPTIONS_URI to the cli service.

### LINKED ISSUE #1114: Add DRUSH_OPTIONS_URI to cli container
**Description**
Setting `DRUSH_OPTIONS_URI` to `http://${VIRTUAL_HOST}` will allow projects using Drush 9 to have the `drush uli` command output a usable URL rather than the current `http://default/...`. Other things like indexing search content, etc, via drush will also then have the proper url.

PR to come.

### PR DESCRIPTION
**Pull request by jhedstrom**
- Closes #1114

### CODE REVIEW

**Comment by lpeabody** [comment]
@jhedstrom is a Drush alias specifies it's own URI, I'm assuming that overrides this option? If so, this addition is rather rad.

**Comment by jhedstrom** [comment]
@lpeabody good question! I'm digging into that and will get back to you. It isn't clear from https://docs.drush.org/en/master/using-drush-configuration/

**Comment by lpeabody** [comment]
Seems like it does override, which would make sense, I was just asking because I'm a paranoid developer.

```
[les.peabody@~/Sites/corvette (feature/core-update)]
-> fin exec 'echo $DRUSH_OPTIONS_URI'
http://domain.docksal

[les.peabody@~/Sites/corvette (feature/core-update)]
-> fin drush @local.com uli
http://com.domain.docksal/user/reset/1/1565310275/MzboziHxPaQYvSQDhKuc29poboV9aJ9bLbBxP-eN7X8/login
```

**Comment by jhedstrom** [comment]
Greg from the drush slack channel also verified this to be the case.

> `DRUSH_OPTIONS_URI` is equivalent to setting the option in a configuration file. Commandline and alias options have a higher priority.

**Comment by sean-e-dietrich** [comment]
@jhedstrom @lpeabody how does this work with multi-sites?

**Comment by jhedstrom** [comment]
@sean-e-dietrich it wouldn't. This is to provide a better default behavior out-of-the-box than `http://default/` for projects that have not yet specified any alias files. Multisite projects would still need to craft alias files I think :)

**Comment by sean-e-dietrich** [comment]
@jhedstrom my suggestion here would be to leave the value empty then and then allow someone to change per project

File: stacks/services.yml [hunk 1]
```diff
@@ -147,6 +147,7 @@ services:
       - XDEBUG_CONFIG=remote_connect_back=0 remote_host=${DOCKSAL_HOST_IP}  # Point xdebug to the host IP
       - PHP_IDE_CONFIG=serverName=${VIRTUAL_HOST}
       - DRUSH_ALLOW_XDEBUG=${XDEBUG_ENABLED:-0}
+      - DRUSH_OPTIONS_URI=http://${VIRTUAL_HOST}
```
Comment (sean-e-dietrich) [inline stacks/services.yml]:
```suggestion
      - DRUSH_OPTIONS_URI
```

Comment (sean-e-dietrich) [reply stacks/services.yml]: that way the variable can be customized

**Comment by jhedstrom** [comment]
These changes have been made

**Comment by sean-e-dietrich** [comment]
hi @jhedstrom took a look at this and appears that we can't do variables in variables which is a bummer. I don't want to set to the virtual host for a fear that for multi-sites. I'm wondering if we were to set this as an empty variable and then allow people to configure per project might be a happy medium. thoughts?

**Comment by lmakarov** [comment]
Without any overrides or mappings in `sites.php`, drush will pick the uri based on the site folder name, which is `default` by default.

```
$ drush --version
Drush Launcher Version: 0.6.0
Drush Commandline Tool 9.7.1

docker@cli:/var/www/docroot$ drush st
 Drupal version   : 8.8.0-dev                                     
 Site URI         : http://default
 ...

docker@cli:/var/www/docroot/sites/site1$ drush st
 Drupal version   : 8.8.0-dev                                     
 Site URI         : http://site1

```

`DRUSH_OPTIONS_URI` only works with Drush 9+ and is an equivalent (almost) of `--uri` command line parameter for Drush 8. `--uri` is also supported with Drush 9 and takes precedence over `DRUSH_OPTIONS_URI` as well as aliases (https://github.com/drush-ops/drush/pull/3966)

Eventually, the order of precedence is the following (from most to least important):

- `--uri` on command line
- `uri` in alias
- `DRUSH_OPTIONS_URI`

```
$ DRUSH_OPTIONS_URI="${VIRTUAL_HOST}" drush st
 Drupal version   : 8.8.0-dev                                     
 Site URI         : http://drupal8-contrib.docksal
 ...

$ drush @example.local st
 Drupal version   : 8.8.0-dev                                     
 Site URI         : http://www.example.com
 ...

DRUSH_OPTIONS_URI="${VIRTUAL_HOST}" drush @example.local st
 Drupal version   : 8.8.0-dev                                     
 Site URI         : http://www.example.com

$ DRUSH_OPTIONS_URI="${VIRTUAL_HOST}" drush @example.local st --uri=local.docksal
 Drupal version   : 8.8.0-dev                                     
 Site URI         : http://local.docksal
 ...
```

Sooo... there is absolutely no hard to multisite projects (or anything else for that matter) in setting the `DRUSH_OPTIONS_URI=${VIRTUAL_HOST}` env variable on the cli container.

**Pull request merged**